src/scripts/main.js

```diff
@@ -1,7 +1,106 @@
 'use strict';
 
-// Uncomment the next lines to use your game instance in the browser
-// const Game = require('../modules/Game.class');
-// const game = new Game();
+const Game = require('../modules/Game.class');
+const field = document.querySelector('.game-field');
+const startBtn = document.querySelector('.start');
+const messages = document.querySelectorAll('.message');
+const scoreDisplay = document.querySelector('.game-score');
 
-// Write your code here
+const STATUS_CLASSES = {
+  start: 'message-start',
+  win: 'message-win',
+  lose: 'message-lose',
+};
+
+const game = new Game();
+let firstMove = true;
+
+startBtn.addEventListener('click', () => {
+  if (startBtn.textContent === 'Start') {
+    game.start();
+
+    showMessage(null);
```

Comment:
you should also update the start button here as well - now I can click start many times until I touch _any_ keyboard button